src/main/java/seedu/address/model/client/Client.java

```diff
@@ -101,6 +102,20 @@ public String getClaimsString() {
         return (numberOfOpenClaims == 0) ? NO_CLAIMS_MESSAGE : String.format(OPEN_CLAIMS_MESSAGE, numberOfOpenClaims);
     }
 
+    /**
+     * Accesses and retrieves a string representation of the claims associated with the current entity.
+     *
+     * This method constructs a message that includes the name of the entity and a predefined access claims message.
+     * It then appends the result of accessing claims from the {@link InsurancePlansManager}.
+     *
+     * @return A string that combines the entity's name, an access claims message,
+     *         and the claims information retrieved from the {@link InsurancePlansManager}.
+     */
+    public String accessClaims() {
+        return this.name.toString() + ACCESS_CLAIMS_MESSAGE
+                    + this.insurancePlansManager.accessClaims();
```

Comment:
is it possible to move this entire function to the execute function to keep consistency with how the rest of the insurance plans and claims methods are currently being used?

Reply:
yup I agree, will make the changes!